tests/integration/targets/cpanm/tasks/main.yml

```diff
@@ -0,0 +1,49 @@
+# (c) 2020, Berkhan Berkdemir
+
+# This file is part of Ansible
+#
+# Ansible is free software: you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation, either version 3 of the License, or
+# (at your option) any later version.
+#
+# Ansible is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with Ansible.  If not, see <http://www.gnu.org/licenses/>.
+
+- name: Instal make, curl and perl
+  package:
+    name:
+      - make
+      - curl
+      - perl
+    state: present
+  become: yes
+
+- name: Install perl development package for Red Hat family
+  package:
+    name: perl-devel
+    state: present
+  when: ansible_os_family == "RedHat"
+
+- name: Install cpanminus
+  raw: "curl -L http://cpanmin.us | sudo perl - --sudo App::cpanminus"
```

Comment:
I would avoid such commands at any costs (especially in tests). Is there a way to install this with the package manager?

Reply:
You are correct. I also didn't want to do it but that day I was focused on making CI green yet failed. I will take a look at it :+1:

Reply:
Thanks! :)

Reply:
May I ask for help because still didn't find "a solution" to install cpanm on CentOS 7.8 and RHEL 8.1.